src/Compilers/CSharp/Test/Emit/Attributes/AttributeTests_WellKnownAttributes.cs

```diff
@@ -8411,6 +8411,112 @@ public static int Main()
 
         #endregion
 
+        #region SkipLocalsInitAttribute
+
+        [Fact]
+        public void SkipLocalsInitAttributeRecognizedAsWellKnownTypeWhenDeclaredInSource()
+        {
+            var source = @"
+namespace System.Runtime.CompilerServices
+{
+    public class SkipLocalsInitAttribute : System.Attribute
+    {
+    }
+}
+";
+
+            var comp = CreateStandardCompilation(source);
+            comp.VerifyDiagnostics();
+
+            var wellKnownType = comp.GetWellKnownType(WellKnownType.System_Runtime_CompilerServices_SkipLocalsInitAttribute);
+            var typeMember = comp.GlobalNamespace
+                .GetNestedNamespace("System")
+                .GetNestedNamespace("Runtime")
+                .GetNestedNamespace("CompilerServices")
+                .GetTypeMember("SkipLocalsInitAttribute");
```

Comment:
There is a helper method to find a nested member: `comp.GetMember<NamedTypeSymbol>("System.Runtime.CompilerServices.SkipLocalsInitAttribute")`.

Reply:
Good suggestion. Thanks!